crates/sc-consensus-subspace/src/archiver.rs

```diff
@@ -377,11 +383,14 @@ pub enum CreateObjectMappings {
 }
 
 impl CreateObjectMappings {
+    /// The minimum mapping block number.
+    pub const MIN_BLOCK: CreateObjectMappings = CreateObjectMappings::Block(NonZeroU32::MIN);
```

Comment:
Nit: not sure this is very helpful, writing `CreateObjectMappings::MIN_BLOCK` is not much shorter than `CreateObjectMappings::Block(NonZeroU32::MIN)` (same in `CreateObjectMappingConfig`)

Reply:
I thought it would be easier if we changed the minimum block in future, but happy to fix it.